db/migrate/20250226225252_remove_site_twitter_sign_in.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class RemoveSiteTwitterSignIn < ActiveRecord::Migration[6.1]
+  def up
+    # Remove site preference for twitter sign in
+    execute "DELETE FROM preferences WHERE owner_type = 'Site' AND name = 'twitter_sign_in'"
```

Comment:
This query will be quite slow due to minimal indices on the preferences table. I don't expect that to be a problem necessarily - I believe only affected rows would get locked after postgres figures out what those are. There will also only be 1 matching row. Not necessary, but a possible change would be to loop through each site's preferences separately as that would leverage the table index, e.g. `Site.all.each{ |site| site.stored_preferences.where( name: "twitter_sign_in" ).destroy_all }`

Reply:
Thanks, I updated it to use the approach you recommended.